Jenkinsfile

```diff
@@ -35,12 +35,27 @@ pipeline {
 
     stage('Scan Secretless') {
       parallel {
-        stage('Scan Secretless Image') {
+        stage('Scan Secretless Images') {
```

Comment:
```suggestion
        stage('Scan Secretless Image') {
```

Reply:
Agreed - not sure what I was thinking there. Thanks!